README.md

```diff
@@ -63,10 +63,10 @@ The start view date. All the dates before this date will be disabled.
 
 The initial value of the start date.
 
-### endDate
+### lastSelectableDate
 
-- Type: `Date` or `String` or `Boolean`
-- Default: `false`
+- Type: `Date` or `String` or `Number`
+- Default: `Infinity`
```

Comment:
**infinity?**

Reply:
The code was default to infinity when forked. It allows the user keep scrolling on the calendar. However, in empire, it has a specified end date of 1 year from today